test/connectObservable.test.tsx

```diff
@@ -1,14 +1,14 @@
+import { fireEvent, render, screen } from "@testing-library/react"
+import { renderHook } from "@testing-library/react-hooks"
```

Comment:
```suggestion
import { renderHook, act } from "@testing-library/react-hooks"
```
lets not forget to import `act` :slightly_smiling_face: